spec/models/ems_event_spec.rb

```diff
@@ -122,6 +122,7 @@
 
       context "messaging_type: artemis" do
         before { stub_settings_merge(:messaging => {:type => 'artemis'}) }
+        before { stub_settings_merge(:workers => {:worker_base => {:queue_worker_base => {:defaults => {:dequeue_method => "miq_messaging"}}}}) }
```

Comment:
Actually sorry I missed this wasn't a `-` on the first line, better to pass both keys to the stub_settings_merge
```suggestion
        before do
          stub_settings_merge(
            :messaging => {:type => 'artemis'}),
            :workers    => {:worker_base => {:queue_worker_base => {:defaults => {:dequeue_method => "miq_messaging"}}}}
          )
        end
```

Reply:
@nasark given the bug we had I think we should add a context here for messaging_type=>[artemis/kafka] but dequeue_method=>drb and add a test to confirm we use miq_queue _not_ manageiq_messaging